modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -42,6 +42,8 @@ GrainTracker::GrainTracker(const InputParameters & parameters) :
 {
   if (!_is_elemental && _compute_op_maps)
     mooseError("\"compute_op_maps\" is only supported with \"flood_entity_type = ELEMENTAL\"");
+
+  _empty_2.resize(_n_vars, libMesh::invalid_uint);
```

Comment:
Right, talk to @dschwen about that choice. At one point in time the grain tracker used 1-based indexing so zero meant "no grain". Somebody asked me to change that too. I'm flexible!